lc0_main.go

```diff
@@ -1074,10 +1080,10 @@ func main() {
 		log.Fatal("You must specify a non-empty password")
 	}
 
-	if *report_host {
+	if *report_host && *localHost == "Unknown" {
```

Comment:
Obviously, the check here should be for empty string.

Reply:
Done